kubernetes/values.yaml

```diff
@@ -34,12 +34,17 @@ securityContext:
 
 # See https://github.com/helm/charts/tree/master/stable/postgresql
 postgresql:
-  postgresqlUsername: kemal
-  postgresqlPassword: kemal
-  postgresqlDatabase: invidious
-  initdbUsername: kemal
-  initdbPassword: kemal
-  initdbScriptsConfigMap: invidious-postgresql-init
+  image:
+    registry: quay.io
```

Comment:
Why quay instead of docker hub?

Reply:
Reviewing the installation instructions the project seemed to have a preference that quay.io was preferred: https://docs.invidious.io/Installation.md 

> The Invidious docker image is only [available on Quay](https://quay.io/repository/invidious/invidious) because, unlike Docker Hub, [Quay is Free and Open Source Software](https://github.com/quay/quay/blob/master/LICENSE). This is reflected in the docker-compose.yml file used in this walk-through.

The helm chart was already pulling invidious from quay.io, so I have kept that approach for pulling this dependency :)